src/core/public/chrome/chrome_service.tsx

```diff
@@ -99,6 +99,12 @@ interface StartDeps {
   workspaces: WorkspacesStart;
```

Comment:
Could we remove the `workspaces` deps here? It seems only been used in collapsibleNavHeaderRender, I think the workspaces menu can `useOpenSearchDashboards` to access the workspaces service.